man/gemfile.5.ronn

```diff
@@ -287,7 +303,17 @@ Git repositories support a number of additional options.
   * `branch`, `tag`, and `ref`:
     You `MUST` only specify at most one of these options. The default
     is `:branch => "master"`
+  * For example:
+
+            `git 'https://github.com/rails/rails.git', :branch => '5-0-stable' do`
+
+            `git 'https://github.com/rails/rails.git', :tag => 'v5.0.0' do`
```

Comment:
These don't need the backticks, ronn files are a bit different from markdown files. See https://rtomayko.github.io/ronn/ or feel free to PM on slack for more information.